test/Semantics/resolve40.f90

```diff
@@ -72,3 +72,9 @@ subroutine s9
   !ERROR: 'i' is already declared in this scoping unit
   data ((x(i,i),i=1,2),i=1,2)/4*0.0/
 end
+
+module m10
```

Comment:
Please ensure that we've got at least one error test that has a namelist name conflicting with a variable (or other symbol), in both orders of declaration.

Reply:
OK, I've add those tests.